external.yml

```diff
@@ -25,30 +25,58 @@
               ext_iface: "{{ ext_interface.stdout }}"
         when: virtual_uc != true
 
+      - name: fail when both public_external_interface and external_network_vlan_id defined
+        fail:
+          msg: Both public_external_interface and external_network_vlan_id defined
+        when: public_external_interface is defined and external_network_vlan_id is defined
+ 
       - name: set ext_iface
         set_fact:
           ext_iface: "{{ vm_external_interface }}"
         when: virtual_uc == true
 
+      # "Reverse path" filter prevents packets from coming in or going out on
+      #  an unexpected interface to help prevent routing loops (check with lab team).
+      # The default route on undercloud was out the primary foreman interface,
+      # so when undercloud gets packets coming in on the 4th NIC, it sends the traffic back
+      # out the foreman interface. Turning off this configuration enables to use
+      # public_external_interface i.e 4th interface
+      - name: enable rp_filter
```

Comment:
This task is disabling rp filter not enabling.

Reply:
agree